CMakeLists.txt

```diff
@@ -90,7 +93,8 @@ set(bot_core_lcmtypes_dependencies lcm)
 set(director_dependencies lcm bot_core_lcmtypes libbot)
 set(iris_dependencies eigen mosek)
 set(signalscope_dependencies director)
-set(drake_dependencies cmake eigen googletest lcm bot_core_lcmtypes libbot bullet octomap snopt gurobi swig_matlab swigmake)
+set(drake_dependencies cmake eigen googletest lcm bot_core_lcmtypes libbot bullet libccd flann fcl octomap snopt gurobi swig_matlab swigmake)
+set(fcl_dependancies flann libccd)
```

Comment:
`dependencies` is misspelled here; as a result, the fcl dependencies are effectively not set